src/strategies/dps-nft-strategy/index.ts

```diff
@@ -0,0 +1,77 @@
+import { getAddress } from '@ethersproject/address';
+import {  subgraphRequest } from '../../utils';
+
+export const author = 'andreibadea20';
+export const version = '0.1.0';
+
+const DPS_SUBGRAPH_URL = {
+  '80001': 'https://api.thegraph.com/subgraphs/name/andreibadea20/subgraph-dps'
+};
+
+const PAGE_SIZE = 1000;
+
+const params = {
+    users: {
+        __args: {
+            first: PAGE_SIZE,
+            skip: 0
+        },
+        id: true,
+        numberOfTokens: true
+    }
+};
+
+const paramsForNFTs = {
+    nftlockeds: {
+        tokenId: true,
+        owner: {
+            id: true
+        },
+        transferedAtTimestamp: true
+    },
+    nftafterClaimeds: {
+        tokenId: true,
+        owner: {
+            id: true
+        },
+        transferedAtTimestamp: true
+    }
+};
```

Comment:
Is there a way to pass `snapshot` into this subgraph? This way it can change time to time and doesn't return same voting power when proposal is open